codegen/grid.py

```diff
@@ -717,6 +720,13 @@ def set_variable(self, var, value=0, type='int', constant=False):
             var = var.name
         self.defined_variable[var] = Variable(var, value, type, constant)
 
+    def set_page_size(self, page_size):
```

Comment:
This should be "set_alignment_size()". We would only want to set the alignment size to PAGESIZE on a ccNUMA system. On other systems such as Xeon Phi the alignment should be the SIMD vector size on the system.